terraform/groups/ecs-service/locals.tf

```diff
@@ -10,7 +10,7 @@ locals {
   kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
   lb_listener_rule_priority  = 63
   lb_listener_paths          = ["/transactions/*/confirmation-statement/*", "/confirmation-statement/*", "/private/confirmation-statement/*", "/private/transactions/*/confirmation-statement/*/filings", "/confirmation-statement-api/healthcheck"]
```

Comment:
think it's this line that you need to remove the healtcheck from, in order for it not to be picked up by the load balancer